.github/workflows/test.yml

```diff
@@ -10,13 +10,15 @@ on:
     - cron:  '0 * * * *'
 
 jobs:
-  build_and_test:
-    runs-on: ${{ matrix.os }}
+  build_and_test_ubuntu:
+    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-          os: [macOS-latest, ubuntu-18.04]
           python-version: [3.5, 3.6, 3.7]
+    container:
+      image: ubuntu:bionic
+      options: -v /var/run/docker.sock:/var/run/docker.sock
```

Comment:
Why is docker socket required?

Reply:
To avoid docker in docker as cross-compile runs Docker tests.

Reply:
Nvm, I misread the repository. Docker socket is required since the tests use docker and Docker in Docker is required.